content/applications/sales/crm/track_leads/lead_scoring.rst

```diff
@@ -1,73 +1,163 @@
-Domain to each team. The Domain can include lead scores.
+- :guilabel:`State`: the geographical state from which the opportunity originates
+- :guilabel:`Country`: the geographical country from which the opportunity originates
+- :guilabel:`Phone Quality`: whether or not a phone number is listed for the opportunity
+- :guilabel:`Email Quality`: whether or not an email address is listed for the opportunity
+- :guilabel:`Source`: the source of an opportunity (e.g. search engine, social media)
+- :guilabel:`Language`: the spoken language specified on the opportunity
+- :guilabel:`Tags`: the tags placed on the opportunity
 
-.. image:: lead_scoring/team-assignation.png
-   :align: center
-   :alt: Team Assignments using domains
+.. note::
+   The variables `Stage` and `Team` are always in effect. `Stage` refers to the CRM pipeline stage
+   that an opportunity is in. `Team` refers to the sales team that is assigned to an opportunity.
+   Predictive lead scoring will *always* take into account these two variables, regardless of which
+   optional variables are selected.
 
-Assign leads to specific salespeople using further refined Domains at :menuselection:`CRM --> Leads
---> Leads Assignment`.
+Next, click on the date next to :guilabel:`Consider leads created as of the:` to select the date
+from which predictive lead scoring will begin its calculations.
 
-.. image:: lead_scoring/lead-assignment-filters.png
-   :align: center
-   :alt: Lead Assignments
+Lastly, click :guilabel:`Confirm` to save changes.
 
-Odoo automatically assigns new leads to teams/salespeople once per day.
+Change the probability manually
+-------------------------------
 
-Evaluate and use the unassigned leads
-=====================================
+The probability of success of an opportunity can be changed manually on the opportunity form. Click
+on the probability number to edit it.
 
-Not all leads will fall into your set Domains and get assigned automatically by Odoo.
+.. important::
+   Manually changing the probability removes the automatic probability updates for that
+   opportunity. The probability will no longer update automatically as the opportunity moves
+   through each stage of the pipeline.
 
-To view your unassigned leads, go to :menuselection:`CRM --> Leads --> Leads` and select the
-Unassigned filter.
+To reactivate automatic probability, click on the gear icon next to the probability percentage.
 
-.. image:: lead_scoring/unassigned-filter.png
+.. image:: lead_scoring/probability-gear-icon.png
    :align: center
-   :alt: Filtering for unassigned leads
+   :alt: The gear icon used to reactivate automatic probability on an opportunity form.
 
-.. note::
-   You can also find unassigned leads and create re-engagement campaigns using the :guilabel:`Email
-   Marketing` or :guilabel:`Marketing Automation` apps.
+Assign leads based on probability
+=================================
+
```

Comment:
consider a brief intro sentence here to break things up a bit, and lead into the next portion of the doc. Just a thought...

Reply:
Good idea, thanks :)